tests/sauceDemo_tests.js

```diff
@@ -0,0 +1,107 @@
+
+
+import { Selector, t, ClientFunction }  from "testcafe";
+
+const baseUrl = 'https://www.saucedemo.com/';
+const validUserName = 'standard_user';
+const validPassword = 'secret_sauce';
+const invalidUserName = 'locked_out_user';
+const invalidPassword = 'not_so_secret_sauce';
+const userNameInput = Selector("#user-name");
+const passwordInput = Selector("#password");
+const loginButton = Selector('#login-button');
+const errorContainer = Selector('.error h3');
+const productSortDrop = Selector('div.right_component > span');
+const sortProductAZ = Selector('option:nth-child(1)');
```

Comment:
Do it better!

Reply:
Modified to const sortProductAZ = Selector('option[value="az"]');